guiEditor/src/sharedComponents/textLineComponent.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";

```

Comment:
should be shared

Reply:
Did this move after your comment, or are you referring to a different directory?